ui/app/components/button-group/button-group.component.js

```diff
@@ -0,0 +1,61 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classnames from 'classnames'
+
+export default class ButtonGroup extends PureComponent {
+  static propTypes = {
+    defaultActiveButtonIndex: PropTypes.number,
+    disabled: PropTypes.bool,
+    children: PropTypes.array,
+    className: PropTypes.string,
+    style: PropTypes.object,
+  }
+
+  static defaultProps = {
+    className: 'button-group',
+  }
+
+  state = {
+    activeButtonIndex: this.props.defaultActiveButtonIndex || 0,
+  }
+
+  handleButtonClick (activeButtonIndex) {
+    this.setState({ activeButtonIndex })
+  }
+
+  renderButtons () {
+    const { children, disabled } = this.props
+
+    return React.Children.map(children, (child, index) => {
+      return child && (
```

Comment:
Do we want to commit to doing either `foo && (<div />)` vs. `foo ? (<div />) : null`?

Reply:
I prefer `&&` since it looks cleaner to me and it's used in the [react docs](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) but I don't mind using `? :` if there are downsides to `&&`